__init__.py

```diff
@@ -44,7 +44,8 @@
             print("The Retriever lacks permission to access the ~/.retriever/ directory.")
             raise
 SCRIPT_SEARCH_PATHS = [
-    os.path.join(HOME_DIR, 'scripts/'),
+    'scripts',
+    os.path.join(HOME_DIR, 'scripts/')
 ]
```

Comment:
@ethanwhite with the json, the only way that a user can create a script is by using the CLI, after which they are stored in .retriever/scripts directory. There does not seem to be any case where a user would want to move a script from this directory to her home directory. Instead, if there are some stray JSON files in the user's home directory, they would get picked up by retriever, and give a compilation error.